### ISSUE TITLE
freetype.PutText not working write text plus diagonal lines (see below)

### ISSUE THREAD

**Issue by peterdn1** [emgucv/emgucv#265]
![Capture](https://user-images.githubusercontent.com/257905/64128282-9154ec00-cd6a-11e9-8fdd-1a0258143d64.JPG)
string fontpath = "D:/FONTS/arial.ttc";// I have tried with other fonts same result.
Freetype2 freetype = new Freetype2();
freetype.LoadFontData(fontpath, 0);     
freetype.PutText(frame, "12345", new Point(25, 100), 12, green, 1, LineType.AntiAlias, false);

**Comment by peterdn1** [comment]
Version#:4.1.0.3420

**Comment by emgucv** [comment]
Tested to work with this unit test:

**Comment by peterdn1** [comment]
I ran the test and it works as expected. I've tried with the font below:
https://www.fontsquirrel.com/fonts/roboto
failed
https://github.com/hirokimoto/Tatabuilder/blob/master/fonts/arial.ttc
failed
![Capture2](https://user-images.githubusercontent.com/257905/64190608-9e85df80-ce2b-11e9-998f-c4a0378d186d.JPG)

Can you add some additional tests?

Thanks,
Peter

**Comment by peterdn1** [comment]
![hello](https://user-images.githubusercontent.com/257905/64190815-0a684800-ce2c-11e9-81c4-88d81ff37b89.JPG)

public void TestFreetype(Mat _frame)
        {
            using (Mat m = new Mat(new Size(640, 480), DepthType.Cv8U, 3))
            using (Emgu.CV.Freetype.Freetype2 freetype = new Freetype2())
            {
                m.SetTo(new MCvScalar(0, 0, 0, 0));
                freetype.LoadFontData("D:/FONTS/arial.ttc", 0);
                freetype.PutText(m, "hello there 123456780 - _ + =", new Point(100, 100), 50, new MCvScalar(255, 255, 0), 1, LineType.EightConnected, false);
                CvInvoke.NamedWindow("test");
                CvInvoke.Imshow("test", m);
                CvInvoke.WaitKey();
            }
        }

**Comment by peterdn1** [comment]
I've tried both 
TrueType 
arail.ttf
TrueType Collection (TTC) 
arial.ttc

I have also tried a number of other fonts always with the same result. I downloaded your font from the test from the following site and it works.
https://github.com/googlefonts/noto-cjk/blob/master/NotoSansCJK-Regular.ttc

I downloaded other fonts from this same site and get the same failure.  Is there an English font that works for you? Any site with a working English font that I can test with?

Thanks,
Peter
